Modules/fact.py

```diff
@@ -0,0 +1,225 @@
+"""
+fact.py - Fact Class
+
+Provides a Fact object for the Fact Manager Class.
+
+Copyright (c) 2018 The Fuel Rat Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+
+import datetime
+
+
+class Fact:
+    def __init__(self, name=None, lang='en', message=None,
+                 aliases=None, author=None, edited=None,
+                 editedby=None, mfd=False):
+        self._name = name
+        self._lang = lang
+        self._message = message
+        self._aliases = aliases
+        self._author = author
+        self._editedby = editedby
+        self._edited = edited if edited else datetime.datetime.now(datetime.timezone.utc)
```

Comment:
It may be wise to actually verify that the `datetime` object passed here is timezone aware.

Reply:
83935a7